src/dmd/opover.d

```diff
@@ -432,6 +432,43 @@ Expression op_overload(Expression e, Scope* sc, TOK* pop = null)
             }
         }
 
+        override void visit(PostExp pe)
+        {
+            AggregateDeclaration ad = isAggregate(pe.e1.type);
+            if (!ad)
+                return visit(cast(BinExp) pe);
+            auto fdr = search_function(ad, Id.opUnaryRight);
+            if (!fdr)
+                return visit(cast(BinExp) pe);
+            
+            void errorRet()
+            {
+                result = ErrorExp.get();
```

Comment:
I know there is a method somewhere that does exactly this but I can't remember whats its called.